bulletroboy/force_decomposition.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from scipy.optimize import minimize
+
+MIN_TENDON_FORCE = 0.01
+MAX_COLLISION_FORCE = 100
+
+def decompose_force_link_to_ef(link_id):
+    """Gets link bounding box dimensions.
+    
+    Args:
+        link_id (int): Index of the link to search.
+
+    Returns:
+        3darray[float]: x, y, and z dimensions of the bounding box.
+
+    """
+    end_effector = None
+    if link_id in [7, 8]:
```

Comment:
please no hardcoded values like these, get it from the robot simulations or parameter server, or config file

Reply:
concerns ids as well as strings